src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutPresenter.cs

```diff
@@ -1,13 +1,958 @@
+				// which retrieves our content as plain text (e.g., if our content is a string,
+				// it returns that; if our content is a TextBlock, it returns its Text value, etc.)
+				// MenuFlyoutPresenterGenerated.GetPlainText(strPlainText);
+
+				// If we get the plain text from the content, then we want to truncate it,
+				// in case the resulting automation name is very long.
+				// Popup.TruncateAutomationName(strPlainText);
+			}
+
+			return automationName;
+		}
+
+		void IMenuPresenter.CloseSubMenu()
+		{
+			var subPresenter = m_wrSubPresenter?.Target as IMenuPresenter;
+
+			if (subPresenter != null)
+			{
+				subPresenter.CloseSubMenu();
+			}
+
+			ISubMenuOwner owner;
+			owner = m_wrOwner?.Target as ISubMenuOwner;
+
+			if (owner != null)
+			{
+				owner.CloseSubMenu();
+			}
+
+			// Reset the focused index not to cached the previous focused index when
+			// the sub menu is opened in the next time
+			m_iFocusedIndex = -1;
+		}
+
+		void DelayCloseMenuFlyoutSubItem()
+		{
+			var subMenuPresenter = m_wrSubPresenter?.Target as IMenuPresenter;
+
+			if (subMenuPresenter != null)
+			{
+				ISubMenuOwner subMenuOwner;
+				subMenuOwner = (subMenuPresenter.Owner);
+
+				if (subMenuOwner != null)
+				{
+					subMenuOwner.DelayCloseSubMenu();
+				}
+			}
+		}
+
+		void CancelCloseMenuFlyoutSubItem()
+		{
+			var subMenuPresenter = m_wrSubPresenter?.Target as IMenuPresenter;
+
+			if (subMenuPresenter != null)
+			{
+				ISubMenuOwner subMenuOwner;
+				subMenuOwner = (subMenuPresenter.Owner);
+
+				if (subMenuOwner != null)
+				{
+					subMenuOwner.CancelCloseSubMenu();
+				}
+			}
+		}
+
+		DependencyObject GetParentMenuFlyoutSubItem(DependencyObject nativeDO)
+		{
+			var spThis = nativeDO as MenuFlyoutPresenter;
+			ISubMenuOwner spResult;
+			spResult = (spThis as IMenuPresenter).Owner;
+
+			var spResultAsMenuFlyoutSubItem = spResult as MenuFlyoutSubItem;
+
+			if (spResultAsMenuFlyoutSubItem != null)
+			{
+				return spResultAsMenuFlyoutSubItem as DependencyObject;
+			}
+			else
+			{
+				return null;
+			}
+		}
+
+		internal void UpdateTemplateSettings()
+		{
+			var templateSettings = TemplateSettings;
+			var templateSettingsConcrete = templateSettings as MenuFlyoutPresenterTemplateSettings;
+
+			var ownerFlyout = GetParentMenuFlyout();
+
+			if (ownerFlyout != null && templateSettingsConcrete != null)
+			{
+				// Query MenuFlyout Content MinWidth, given the input mode, from resource dictionary.
+				var resources = Resources;
+				var flyoutContentMinWidth = (double)resources[
+	(ownerFlyout.InputDeviceTypeUsedToOpen == FocusInputDeviceKind.Touch || ownerFlyout.InputDeviceTypeUsedToOpen == FocusInputDeviceKind.GameController)
+		? "FlyoutThemeTouchMinWidth" : "FlyoutThemeMinWidth"];
+				var visibleBounds = this.LayoutSlotWithMarginsAndAlignments;
+				// DXamlCore.GetCurrent().GetVisibleContentBoundsForElement(GetHandle(), &visibleBounds);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)